README.md

```diff
@@ -9,6 +9,7 @@ WABT (we pronounce it "wabbit") is a suite of tools for WebAssembly, including:
  - [**wasm2wat**](https://webassembly.github.io/wabt/doc/wasm2wat.1.html): the inverse of wat2wasm, translate from the binary format back to the text format (also known as a .wat)
  - [**wasm-objdump**](https://webassembly.github.io/wabt/doc/wasm-objdump.1.html): print information about a wasm binary. Similiar to objdump.
  - [**wasm-interp**](https://webassembly.github.io/wabt/doc/wasm-interp.1.html): decode and run a WebAssembly binary file using a stack-based interpreter
+ - [**wasm-decompile**](https://webassembly.github.io/wabt/doc/wasm-decompile.1.html): decompile a wasm binary into readable C-like syntax.
```

Comment:
Looks like these html files are stored in the `gh-pages` branch.     I guess we should have some way to automatically update them?  But I know nothing of this..

Reply:
yeah not sure how this is supposed to work.. I guess @binji knows.

Reply:
Nope, I think it's manual for now. Would be nice to make it automatic though.

Reply:
Github supports to use a sub directory of master branch instead of gh-pages branch now. Then you need not to sync it.

Reply:
Sweet. That sounds way better.

Reply:
Yes, in fact for `docs/` it can be fully automatic, that's why I put this file there.